app/views/users/index.html.erb

```diff
@@ -1,7 +1,6 @@
 <h1><%= t(".title") %></h1>
 <ul>
   <% @users.each do |user| %>
-    <%= render "detail", user: user %>
-    </br>
+    <%= render "user" %>
```

Comment:
I believe you can remove lines 3-5 and just use `<%= render @users %>`

Reply:
Indeed I could - done sha: b3e01ba